sdc/tests/tests_perf/test_perf_utils.py

```diff
@@ -195,13 +195,78 @@ def get_times(f, *args, iter_number=5):
     return exec_times, boxing_times
 
 
+class ResultsDriver:
+    """Base class. Load and dump results."""
+
+    def __init__(self, file_name, raw_file_name=None):
+        self.file_name = file_name
+        self.raw_file_name = raw_file_name if raw_file_name else f'raw_{file_name}'
+
+
+class ExcelResultsDriver(ResultsDriver):
+    # openpyxl need to be installed
+
+    def dump_grouped_data(self, grouped_data, logger=None):
+        try:
+            with pandas.ExcelWriter(self.file_name) as writer:
+                grouped_data.to_excel(writer)
+        except ModuleNotFoundError as e:
+            if logger:
+                msg = 'Could not dump the results to "%s": %s'
+                logger.warning(msg, self.file_name, e)
+
+    def dump_test_results_data(self, test_results_data, logger=None):
+        try:
+            with pandas.ExcelWriter(self.raw_file_name) as writer:
+                test_results_data.to_excel(writer, index=False)
+        except ModuleNotFoundError as e:
+            if logger:
+                msg = 'Could not dump raw results to "%s": %s'
+                logger.warning(msg, self.raw_file_name, e)
+
+    def load(self, logger=None):
+        raw_perf_results_xlsx = Path(self.raw_file_name)
+        if raw_perf_results_xlsx.exists():
```

Comment:
Let's decrease indentations via the following construction:
```
if not raw_perf_results_xlsx.exists():
    # maybe log here the fact that file doesn't exist
    return None
# remaining code
```

Reply:
I know this trick. This code is copied from the original.